plugins/inputs/postgresql/postgresql_test.go

```diff
@@ -21,7 +21,7 @@ func launchTestContainer(t *testing.T) *testutil.Container {
 			"POSTGRES_HOST_AUTH_METHOD": "trust",
 		},
 		WaitingFor: wait.ForAll(
-			wait.ForLog("database system is ready to accept connections"),
+			wait.ForLog("database system is ready to accept connections").WithOccurrence(2),
```

Comment:
Could you add a comment here explaining why its expected for this message to occur twice? (same as you wrote in the PR description)

Do you know if there is a chance for the log message "database system is ready to accept connections" to occur less then twice?

Reply:
done

My preference is for it to show up once, however, there doesn't appear to be a configuration setting to let us do it that way that I noticed.